### DISCUSSION TITLE
'person health care provider role'

### DISCUSSION THREAD

**Discussion by sjbost** [ufbmi/OMRSE#274] [Ideas]
This class came up on last week's OBIB call. Liz was looking for an ontology with 'health care provider role' and was looking at classes available in OMRSE. 

She discovered 'person health care provider role' which has no annotations and is a subclass of 'human social role.' 

She ultimately decided to use 'health care provider role' which is a subclass of 'human health care role' and edited by Bill and Mathias. 

Any idea what is going on with 'person health care provider role'?

**Comment by CDowland** [comment]
The parent class for 'human social role' is 'organism social role,' which includes the following comment in its annotations:

“Defined class that we will ultimately move to an application ontology.  We are leaving here for now until we determine which application ontology: it is likely going to be an ontology that does not currently (2012-06-05) exist.
Ditto for its current descendants.”

I realize that doesn't answer your question, but it's potentially relevant. I recently noticed that comment while looking into something else and have been wondering what the status of this group of classes is (whether they're still intended to be used elsewhere, and whether we still want them in OMRSE).

**Comment by sjbost** [comment]
That's actually helpful as it explains what is going on with 'person health care provider role'. However, that annotation is from 2012 so I'm wondering what the status is of that proposed application ontology and if we can spin off these classes into a module.

**Comment by LightGrenadier** [comment]
#142

**Comment by LightGrenadier** [comment]
Next steps: Sort out immediately surrounding hierarchy around person health care provider role and propose a definition for person health care provider role.

**Comment by hoganwr** [comment]
Note that in our discussion the question arose as to whether "person" was modifying the role or the healthcare provided.  This class was intended from the outset to be a role whose bearer is a person.  The goal was to distinguish it from healthcare provider roles borne by organizations.

For example "Dr. Jone's role as a physician with ABC group practice" is an instance of this class.  The role of "ABC group practice" as an outpatient clinic is an instance of "organization healthcare provider role".

**Comment by LightGrenadier** [comment]
@dillerm will reach out to @CDowland for next steps.

**Comment by CDowland** [comment]
I've reviewed the notes from when I was working on this a while back and tried to summarize the key points. I'm attaching a Word document because copying/pasting the formatted hierarchy here does not turn out well. I hope you'll find some of the suggestions helpful. 
[OMRSE roles hierarchy; person health care provider role.docx](https://github.com/ufbmi/OMRSE/files/9759795/OMRSE.roles.hierarchy.person.health.care.provider.role.docx)

**Comment by hoganwr** [comment]
Responding to Clint's document:

1. "human healthcare role" is not defined as specific to providing healthcare to humans. That label is misleading, and I propose we elevate its alternative label 'health care role' to label, and either remove the current label 'human healthcare role' or demote it to alternative label.
2. Therefore, keeping "patient role" and "human patient role" and their subclasses as defined, as is, is appropriate (I note that the definitions that say "organism" are under the former, and the classes that say either "human" or "Homo sapiens" are under the latter. 
3. We don't have explicit veterinary classes yet because no one has needed them
4. The four classes currently under "healthcare provider role" should be moved under "person healthcare provider role", or per discussion #258, the proposed new class. Probably a new class should be created and "person healthcare provider role" deprecated since it currently has no definition and is ambiguous.

**Comment by hoganwr** [comment]
I converted this issue to a discussion today during my meeting with Matt, because actionable specific items are not coming out of it that can resolve it completely. We will use this discussion thread to "spin off" actionable issue items.
